dev-requirements.txt

```diff
@@ -12,7 +12,7 @@ towncrier==23.6.0
 pytest-memray==1.7.0;python_version<"3.14" and sys_platform!="win32" and implementation_name=="cpython"
 trio==0.26.2
 # https://github.com/pallets/quart/pull/369
-Quart @ git+https://github.com/pallets/quart@5f9d977cfbcaf2844efb5309b0c80a4289af1958
+Quart==0.19.9
```

Comment:
Quart 0.19.9 doesn't include https://github.com/pallets/quart/pull/369 needed for successful CI runs with Python 3.14. Please pin it to a more recent commit like 67110bf383d8973bce1619e957b4b6ea088ad9f2 for now

Reply:
```suggestion
Quart @ git+https://github.com/pallets/quart@67110bf383d8973bce1619e957b4b6ea088ad9f2
```